modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-cert-manager-verify_{context}"]
+== Verify cert-manager installation
+
+To verify cert-manager is installed, follow these steps:
+
+. Check `istiod` certificate:
++
+[source, terminal]
+----
+kubectl exec "$(kubectl get pods -n bookinfo -l app=productpage -o jsonpath='{.items[0].metadata.name}')" -c istio-proxy -n bookinfo -- \
+    openssl s_client -CAfile /var/run/secrets/istio/root-cert.pem -showcerts -connect istiod-basic.istio-system:15012 </dev/null
+----
+
+. Check app certificate:
++
+[source, terminal]
+----
+kubectl exec "$(kubectl get pods -n bookinfo -l app=productpage -o jsonpath='{.items[0].metadata.name}')" -c istio-proxy -n bookinfo -- \
+    openssl s_client -CAfile /var/run/secrets/istio/root-cert.pem -showcerts -connect details.bookinfo:9080 </dev/null
+----
+//Waiting on Jacek/devs to clarify what "mounts" and "root configmap" and "root of trust" mean
+. Verify by seeing all mounts are correct on pilot, launching bookinfo and checking.
+
+. Verify that the right root config map is used for the root of trust.
++
+[source, yaml]
+----
+  security:
+    certificateAuthority:
+      cert-manager:
+        address: cert-manager-istio-csr.cert-manager.svc:443
+        pilotSecretName: istiod-tls
+        rootCAConfigMapName: istio-ca-root-cert
```

Comment:
Please remove fields `pilotSecretName` and `rootCAConfigMapName` as they are deprected.